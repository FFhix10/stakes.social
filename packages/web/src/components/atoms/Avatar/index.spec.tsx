import React from 'react'
import { render } from '@testing-library/react'
import { Avatar } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')
jest.mock('@dev/graphql')

describe(`${Avatar.name}`, () => {
  test('Snapshot', () => {
    const url = 'http://dummy.org'
    const component = render(<Avatar url={url} size="20" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
