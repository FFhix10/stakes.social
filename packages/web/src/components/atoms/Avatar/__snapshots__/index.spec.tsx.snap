// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-2tazhb-0 xkRoZ"
    >
      <img
        alt="avatar"
        src="http://dummy.org"
        style="height: 20px; width: 20px;"
      />
    </div>
  </div>
</body>
`;
